editor/code_editor.h

```diff
@@ -202,6 +202,8 @@ class CodeTextEditor : public VBoxContainer {
 	static void _bind_methods();
 
 	bool is_warnings_panel_opened;
+	bool defaultHighliteCurrentLine;
```

Comment:
Variable names should use `snake_case`, e.g. `default_highlite_current_line`.
Also, there is a typo, so it should be `default_highlight_current_line`

Reply:
About the skipping of comments in the code editor or not, if that is done in GdScript or the GdScript compiler, agree but it would be much more effort so we chose the shorter way to get there.